test/mailers/user_mailer_test.rb

```diff
@@ -22,6 +23,11 @@ def setup
                    has requested to join your team #{user_requests(:request_one).team.team_name}
                    Click the link below to view and accept or reject the request.
                    #{link_to 'View Team Dashboard', team_url(user_requests(:request_one).team)}").squish
+    @remind_email_body = strip_tags("Hello #{users(:user_one).full_name}! This is a reminder for the
+                   upcoming MITRE CTF which will start at #{games(:mitre_ctf_game).start}, which is in
```

Comment:
May want to render the `, which is in` part in javascript as well so that it doesn't show up weird for users without javascript.